backend/dal/Services/ContactService.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
```

Comment:
Note that this isn't hooked up in this PR, I have addressed that in the changes that follow this PR.